src/coreclr/debug/daccess/request.cpp

```diff
@@ -2711,19 +2686,7 @@ ClrDataAccess::GetAssemblyName(CLRDATA_ADDRESS assembly, unsigned int count, _In
     }
     else if (!pAssembly->GetPEAssembly()->IsReflectionEmit())
```

Comment:
Is this DAC API used by anything? It does the same thing as `GetAssemblyLocation`, just the error handling is a bit different, and does not actually return an assembly name as its name says.

IIRC, this API was originally part of an intern project that never shipped.